lib/eventq/eventq_aws/aws_eventq_client.rb

```diff
@@ -100,6 +100,9 @@ def with_prepared_message(event_type, event, context)
         qm.type = event_type
         qm.context = context
         qm.content_type = event.class.to_s
+        if event.respond_to? :Correlation
```

Comment:
I'm curious why it's `:Correlation` with an uppercase C, instead of the usual Ruby style lowercase / snake_case?

Reply:
Below is an example of the structure for the existing event object object:

`{
	"Id": "2e567a40-cfd1-4d44-884d-f0c4d13e0e55",
	"SourceRecordId": "00000000-0000-0000-0000-000000000000",
	"DestinationRecordId": "00000000-0000-0000-0000-000000000000",
	"BootstrapId": "00000000-0000-0000-0000-000000000000",
	"BootstrapPipeline": null,
	"TenantId": null,
	"PublishedAt": "2021-07-20T14:44:06.689+00:00",
	"ReceivedAt": "2021-07-20T14:44:06.6909769+00:00",
	"OccurredAt": null,
	"CorrelationTraceId": "c0258687-c1a3-4bd5-a078-ca8786f0ae05", <<<<<<<<<<<<<<<<<<<<<<<<< Trace 1
	"RawContent": {
		"^o": "S1CentralEvents::Data::MaxwellEvent",
		"database": "business",
		"Correlation": {
			"Trace": "bf78107f-d9ad-470a-ba46-0c00c366df0c" <<<<<<<<<<<<<<<<<<<<<<<<< Trace 2
		},
		"table": "S1Business::Entities::BusinessSetting",
		"type": "update",
		"ts": 1626792246,
		"commit": true,
		"data": {
			"business_id": "ed878235-7ec4-4888-a8fa-f8022626d659",
			"key": "company_tax_details",
			"value": {
				"tax_number": ""
			},
			"created": "2021-07-19T07:42:15+00:00",
			"updated": "2021-07-20T14:44:06+00:00"
		},
		"old": "",
		"environment": "qa7",
		"landscape": "eu"
	},
	"Aborted": false,
	"Ignored": false,
	"Terminated": false,
	"ExecutionAttempts": 1,
	"IsDuplicate": false,
	"ParserType": "SBC.Utility.Datasync.Service.EventWorker.Parsers.MaxwellParser, SBC.Utility.Datasync.Service.EventWorker, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
	"Pipeline": null,
	"Stream": null,
	"Action": 0,
	"VerificationRequestDetails": null
}`

The key `Correlation` exists in the event payload

Reply:
Looks like we're stuck with it then 😩